WalletWasabi.Fluent/ViewModels/CoinControl/CoinSelectorViewModel.cs

```diff
@@ -85,9 +85,15 @@ public CoinSelectorViewModel(WalletViewModel walletViewModel, IList<SmartCoin> i
 		TreeDataGridSource = CoinSelectorDataGridSource.Create(_itemsCollection);
 		TreeDataGridSource.DisposeWith(_disposables);
 
-		RefreshFromPockets(sourceItems);
-		UpdateSelection(coinItemsCollection, initialCoinSelection);
-		ExpandSelectedItems();
+		wallet.Coins.Pockets.ToCollection()
+							.Take(1)
```

Comment:
@ichthus1604 

This Take(1) means it will take 1 pocket only! That's why I see only 1 pocket

Reply:
`ToCollection()` returns `IObservable<IReadOnlyCollection<Pocket>>` and then `Take(1)` means "I only want to listen to the first occurrence of this observable sequence". It's not specifying how many elements from the resulting `IReadonlyCollection` we want to keep

Reply:
@soosr okay I'm very confused here.

Removing `Take(1)` seems to have fixed the issue effectively, but now I'm lost because my understanding is per my previous comment.

Maybe our in-house Rx expert, @SuperJMN can shed some light here? Why does removing `Take(1)` make it work correctly? That's trying to replicate [this](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/ViewModels/CoinControl/CoinSelectorViewModel.cs#L88C7-L88C7), which fires only once for the initial refresh

Reply:
It seems to me `ToCollection()` projectiles the items one by one to the sequence.
So the first sequence will have 1 pocket in it, second sequence two pockets, etc.

That is why I have seen only 1 pocket, because it added them one by one, but it also canceled the subscription after the first sequence.

Reply:
@soosr your observation is the same as mine, but that is not the documented behavior of the `ToCollection()` method, which is why I'm confused about it.

Reply:
Guys, ToCollection materializes the observable changeset to a collection **on every change**. The behavior you're getting here is the expected.

Please, take a look to those 2 pipelines that are quite similar:

**a)** 

```csharp
new[] { 1, 2, 3}
	.AsObservableChangeSet()
	.ToCollection()
	.Dump();
```
That results in this:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/3109851/c9373ba1-275d-4851-b932-13e59e771008)

**b)**

```csharp
new[] { 1, 2, 3 }.ToObservable()
.ToObservableChangeSet()
.ToCollection()
```
That results in this:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/3109851/cdac81b3-5133-4d8f-a846-fc393c343e6d)


a) is using an extension method that send the array as a single change. This is handy, but less flexible.
b) is transforming the array to a sequence, so the "chunk" nature of it is lost. Hence, each element is pushed, like they didn't come as a pack.

Regarding performance, b) is worse. But that's no problem at all if we apply a Throttle.

I think the code is already OK. It has a good balance between readability and performance.